corehq/apps/app_manager/tests/test_suite.py

```diff
@@ -89,8 +90,11 @@ def test_owner_name(self, *args):
     def test_printing(self, *args):
         self._test_generic_suite('app_print_detail', 'suite-print-detail')
 
+    @flag_enabled('MOBILE_UCR')
```

Comment:
This was required for v2 because the v2 report instance `commcare:reports` requires access to the flag [here](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/suite_xml/post_process/instances.py#L380)